backend/app/cmd/server.go

```diff
@@ -198,8 +198,10 @@ type NotifyGroup struct {
 		API     string        `long:"api" env:"API" default:"https://api.telegram.org/bot" description:"telegram api prefix"`
 	} `group:"telegram" namespace:"telegram" env-namespace:"TELEGRAM"`
 	Email struct {
-		From                string `long:"fromAddress" env:"FROM" description:"from email address"`
-		VerificationSubject string `long:"verification_subj" env:"VERIFICATION_SUBJ" description:"verification message subject"`
+		From                 string `long:"fromAddress" env:"FROM" description:"from email address"`
+		VerificationSubject  string `long:"verification_subj" env:"VERIFICATION_SUBJ" description:"verification message subject"`
+		AdminNotifications   bool   `long:"notify_admin" env:"ADMIN" description:"notify admin on new comments via ADMIN_SHARED_EMAIL"`
+		AdminNotifyOnReplies bool   `long:"notify_admin_on_replies" env:"ADMIN_ON_REPLIES" description:"notify admin on replies to comments as well (default is only to top-level comments)"`
```

Comment:
Is this one really needed? I mean all-or-none sounds simple and predictable, but adding an option to notify on replies in some conflict with the regular notification.  Fot instance if admin want to get notifications on all new comments and replies to his comments he need to follow two different paths. My suggestiong is to keep AdminNotifications only and send everyting

Reply:
I've tried making this work for small and not-so-small sites. I think you are right and it's better to have a simple scheme which will work for small sites well without trying to predict potential users desires.